languagetool-core/src/main/java/org/languagetool/tools/StringTools.java

```diff
@@ -232,6 +227,15 @@ public static String lowercaseFirstChar(String str) {
     return changeFirstCharCase(str, false);
   }
 
+  /**
+   * Return <code>str</code> if str is not abbreviation, otherwise return
+   * modified <code>str</code> so that its first character is now a lowercase character.
+   */
+  public static String lowercaseFirstCharIfNotAbbreviation(String str) {
```

Comment:
I'm not sure yet. Could you make the Javadoc describe how "abbreviation" is defined? It looks to me as if this is actually a test for an acronym, i.e. an all-uppercase abbreviation like NATO, NASA etc?

Reply:
Thanks for the notice. We've reworked idea and now will lowercase word only if it is capitalized.